src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/ECCurve.cs

```diff
@@ -194,10 +194,14 @@ public void Validate()
                     throw new CryptographicException(SR.Cryptography_InvalidECNamedCurve);
                 }
 
-                if (Oid == null ||
-                    (string.IsNullOrEmpty(Oid.FriendlyName) && string.IsNullOrEmpty(Oid.Value)))
+                if (Oid == null)
```

Comment:
I'd rather continue to throw CryptographicException here. changing to ArgumentNullException would be a breaking change (which we sometimes make) but changing to ArgumentException would definitely be unacceptable.